management-microservice/src/main/java/com/nearsoft/fgaribay/mgmt/ProductRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.nearsoft.fgaribay.mgmt;
+
+import com.nearsoft.fgaribay.mgmt.model.Product;
+import com.nearsoft.fgaribay.mgmt.model.ProductEntity;
+import org.springframework.data.repository.Repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public interface ProductRepository extends Repository<ProductEntity, Long> {
```

Comment:
If you're implementing manually this repository interface do you really have to extend from org.springframework.data.repository.Repository? Would it make any difference not to?

Reply:
Thanks for approving. I will use the cleaner JPA way that you mentioned on the call, and it should be there on the next pull request.